libvast/src/detail/installdirs.cpp.in

```diff
@@ -60,6 +60,8 @@ std::filesystem::path install_datadir() {
 std::filesystem::path install_configdir() {
 #if VAST_ENABLE_RELOCATABLE_INSTALLATIONS
   const auto [prefix, _] = install_prefix_and_suffix();
+  if (prefix == "/usr")
+    return "/@CMAKE_INSTALL_SYSCONFDIR@/vast";
```

Comment:
Usually this is solved at the configury-level (ie. debian packages are usually built with `--prefix=/usr --sysconfdir=/etc`)

Reply:
This is for relocatable binaries, which do not have build-time configured paths.

Reply:
It seems strange to use configuration time install paths in a relocatable binary, do we even set them?
```suggestion
    return "/etc/vast";
```

Reply:
We use them as relocatable paths, e.g., on some systems lib64 is used over lib and we want to support relocatable binaries for those systems as well.

Reply:
Do we actually *want* this special case if the user has specified a custom SYSCONFDIR? The logic from the PR title (prefix `/usr` -> sysconfdir `/etc`) is now subtly different than whats implemented (prefix `/usr` -> prefix is ignored for sysconfdir)

Reply:
Good point. So probably hardcoded /usr -> /etc it is.

Reply:
I don't think it's possible to know whether to use `lib` or `lib64` at configuration time either. We explicitly support "relocating" the binary to other systems after all.

Reply:
> I don't think it's possible to know whether to use `lib` or `lib64` at configuration time either. We explicitly support "relocating" the binary to other systems after all.

You're misunderstanding. I was saying that we do this relative dir magic in order not to hardcode libdir etc, because they may be relative paths that are not the defaults.